addons/calendar/models/calendar.py

```diff
@@ -159,7 +159,7 @@ def create(self, vals_list):
     def copy(self, default=None):
         raise UserError(_('You cannot duplicate a calendar attendee.'))
 
-    def _send_mail_to_attendees(self, template_xmlid, force_send=False, force_event_id=None):
+    def _send_mail_to_attendees(self, template_id, force_send=False, force_event_id=None):
```

Comment:
I would name the parameter template (or mail_template), as it is not an ID anymore. You can also update the method docstring as template_xmlid is not correct anymore :) .